util/design/lib/LcStEnc.py

```diff
@@ -181,14 +182,24 @@ def _validate_tokens(config):
     '''Validates and hashes the tokens'''
     config.setdefault('token_size', 128)
     config['token_size'] = check_int(config['token_size'])
+    # This needs to be byte aligned
+    if config['token_size'] % 8:
+        raise RuntimeError('Size of token {} must be byte aligned'
```

Comment:
```suggestion
        raise ValueError('Size of token {} must be byte aligned'
```

Would be more typical, but doesn't really matter here.

Reply:
Thanks, I've changed this to `ValueError`